### PR TITLE
fix malformed presets

### PR DESCRIPTION
**Pull request by hunterk**
that RetroArch accepts but choke on more strict parsing, as per https://github.com/libretro/slang-shaders/issues/369.

@chyyran if you can give me a list of presets that erroneously use semicolons at the end of lines, I'm happy to clean those up, as well.

For the record, I believe most of these pass because RetroArch's preset parser just ignores any options/values it doesn't understand, and it evidently doesn't check LUT paths at all and just passes them to the shader parser to accept or choke on.

The missing last pass on geom-deluxe was actually intentional to keep it as in-line with the upstream version in MAME as possible, even if the actual pass didn't exist, but I went ahead and commented it to be more clear instead of relying on the number of passes to keep it legal.

The Mega Bezel super-xbr-crt shader is still broken on my end, but at least its preset is right now. :)

### CODE REVIEW

**Comment by chyyran** [comment]
To the best of my knowledge the presets with semicolons in them are third party and not in slang-shaders. I re-ran the tests without those parsing hacks against the current master to confirm.

**Comment by hizzlekizzle** [comment]
ha, not even all of them, just some. weird!

Anyway, I cleaned those up, too. Do you want to do a quick parse through my fork repo and see if that gets everything cleared up? if so, I'll squash-and-merge here. If not, I'll clean up whatever turns up :)

File: crt/crt-torridgristle.slangp [hunk 1]
```diff
@@ -33,12 +33,12 @@ overlay = ../reshade/shaders/blendoverlay/shadowmask-4x4.png
 filter_overlay = linear
```
Comment (chyyran) [inline crt/crt-torridgristle.slangp:33]: This should be `overlay_linear = true`

**Comment by chyyran** [comment]
Just one left

```
Could not parse ..\test\shaders_slang\crt\crt-maximus-royale.slangp: IOError("\\\\?\\F:\\coding\\librashader\\test\\shaders_slang\\crt\\shaders\\crt-maximus-royale\\FrameTextures\\16_9\\TV_decor_1.png", Os { code: 2, kind: NotFound, message: "The system cannot find the file specified." })
```

**Comment by chyyran** [comment]
Also, `misc\shaders\glass.slang` still has an incorrect include path for `param_floats.h`.

**Comment by hizzlekizzle** [comment]
I'm heading to bed, but if this looks good to you, I'll merge it first thing in the morning. Thanks for your help combing over these!

**Comment by chyyran** [comment]
lgtm 🚀

**Pull request merged**